crates/libcontainer/src/hooks.rs

```diff
@@ -27,16 +27,27 @@ pub enum HookError {
     MissingContainerState,
     #[error("failed to write container state to stdin")]
     WriteContainerState(#[source] std::io::Error),
+    #[error("io error")]
+    OtherIO(#[source] std::io::Error),
```

Comment:
I don't see where this is used.

Reply:
https://github.com/containers/youki/pull/2780/commits/4a4975954e4c8ee1b88c176789225eef37f7a48a